src/pages/race.astro

```diff
@@ -0,0 +1,153 @@
+---
+import Layout from "../layouts/Layout.astro";
+import JoinCards from "../components/JoinCards.astro";
+
+
+import { Image } from "astro:assets";
+import teamImage from "../data/images/team_4.jpg";
+import bunny from "../data/images/bunny.png";
+import carimage from "../data/images/carmolecule.png";
+import typesofcars from "../data/images/typesofcars.png";
+---
+
+</main>
+<Layout title="The Next Race">
+    <main>
+      <Image
+        src={bunny}
+        alt="Formula Nano Team Members on a sunny day standing in front of University of Waterloo Quantum Nano Center"
+        class="mt-8 h-48 w-full rounded-2xl object-cover sm:h-[30rem] object-top"
+      />
+      <h1 class="text-center py-8 text-5xl font-bold">Our Race to the Next Nanocar Competition!</h1>
+    
+
+      <section class="mt-2 space-y-6 text-lg">
+        <p>
+          Formula Nano is gearing up to join the next International Nanocar Race, 
+          an event where teams from around the world control molecule-sized “cars” to compete on the nanoscale!  
+        </p>
+
+        <p>
+          We’re excited to take on the challenge, meet the race’s unique criteria, 
+          and showcase to the world our own molecular-vehicle designs, made with precision and innovation.  
+        </p>
+
+        <p class="text-center">
+          <b>Stay tuned as we share updates on our progress—this is just the beginning!</b>
+        </p>
+  
+        <Image
+          src={carimage}
+          alt="Car and molecule combination picture"
+          class="mt-8 h-64 w-96 rounded-2xl object-cover mx-auto shadow-lg transform hover:scale-105 transition-all duration-300"
+        />
+      </section>
+
+       <!-- Competition Rules Section -->
+    <section class="mt-8 bg-gradient-to-r from-red-100 to-yellow-100 py-12 px-6 sm:px-12 rounded-2xl object-cover shadow-xl hover:scale-105 transition-all duration-300">
```

Comment:
What's the deal with the scaling on hover?

Reply:
it makes the images "pop out"/larger when hovered